index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" type="text/css" href="style.css"> <title>personal </title>
+   <title>trial exercise</title>
+</head>
+<body> 
+  <nav id="navbar">
+      <ul>
+        <li><a href="#about">About</a></li>
+          <!-- <li><a href="#home">home</a></li> --> 
```

Comment:
please do not leave unused code as a comment. un used comment makes your code unclear.

Reply:
[OPTIONAL] I think it is a good idea to add some emojis in your inline comments to make your comments more human-friendly. :space_invader: :robot: Kindly remember to check your code for typos :eyes: :eyes: